specs/core/0_beacon-chain.md

```diff
@@ -1179,8 +1180,8 @@ def get_initial_beacon_state(initial_validator_deposits: List[Deposit],
             withdrawal_credentials=deposit.deposit_data.deposit_input.withdrawal_credentials,
             randao_commitment=deposit.deposit_data.deposit_input.randao_commitment
         )
-        if state.validator_registry[index].balance >= MAX_DEPOSIT * GWEI_PER_ETH:
-            update_validator_status(state, index, ACTIVE)
+        if state.validator_registry[validator_index].balance >= MAX_DEPOSIT * GWEI_PER_ETH:
+            update_validator_status(state, validator_index, ACTIVE)
 
     # set initial committee shuffling
     initial_shuffling = get_new_shuffling(ZERO_HASH, initial_validator_registry, 0)
```

Comment:
looks like an indentation error as well

Reply:
sorry, what's the indentation error do you mean?

Reply:
nevermind, I think I was hallucinating